task/TV3-167 - Update TODOv3 comments

shayanaijaz opened the pull request:
## Overview

Updated TODOv3 comments and added tasks in Jira 

## Related

* [TV3-167](https://jira.tacc.utexas.edu/browse/TV3-167)

## Changes

No code changes, just changes to comments. 

Other than the tasks that already existed the following Jira tasks were added: 

[WP-96](https://jira.tacc.utexas.edu/browse/WP-96) - Cleanup StorageSystem class
[WP-97](https://jira.tacc.utexas.edu/browse/WP-97) - Update TestAgaveUtils
[WP-98](https://jira.tacc.utexas.edu/browse/WP-98) - Update Toast component tests
[WP-99](https://jira.tacc.utexas.edu/browse/WP-99) - Update maxRunTime field formatting for a job submission
[WP-100](https://jira.tacc.utexas.edu/browse/WP-100) - Display all job attributes via getJobDisplayInformation
[WP-101](https://jira.tacc.utexas.edu/browse/WP-101) - Create standalone test util for testing valid user credentials on a system

## Testing

Nothing to test for this

nathanfranklin reviewed (approved):
LGTM 👍 
 
Having the Jira issue with the comment is GREAT. 💯 

---

just left a comment about matlab/license on slack.

rstijerina reviewed (commented):
Looks good! For the remaining untouched TODOv3 comments, do we want to remove some of those @shayanaijaz ?

shayanaijaz commented:
> Looks good! For the remaining untouched TODOv3 comments, do we want to remove some of those @shayanaijaz ?

I added another task [WP-102](https://jira.tacc.utexas.edu/browse/WP-102) based on the comments that I missed and added Jira task numbers to some more comments. I believe all the TODOv3 comments should be covered now.

rstijerina commented on client/src/components/Applications/AppForm/AppForm.jsx:
```diff
@@ -660,7 +660,7 @@ export const AppSchemaForm = ({ app }) => {
                         name="archiveSystemDir"
                         type="text"
-                        placeholder="HOST_EVAL($WORK)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}" // TODOv3: How do we know if portal supports HOME vs WORK?
+                        placeholder="HOST_EVAL($WORK)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}"
```
Actually I think we may want to keep this comment and task. The task should be to determine a safe root path from which we can archive. Perhaps using the `homeDir` of the `defaultSystem`

shayanaijaz replied:
I'll add that back in and create a task for it as well

shayanaijaz replied:
Created [WP-103](https://jira.tacc.utexas.edu/browse/WP-103)
